Makd.mak

```diff
@@ -662,6 +662,13 @@ $O/doc.stamp: $(call find,$(SRC),-type f \( -name '*.d' -o -name '*.di' \))
 
 doc += $O/doc.stamp
 
+# General rule to run the harbored-mod generator
+$O/dfmt.stamp: $(shell find $(SRC) -type f \( -name '*.d' -o -name '*.di' \))
+	$(call exec,dfmt -i $(SRC) >/dev/null,fmt)
+	$Vtouch $@
+
+dfmt += $O/dfmt.stamp
```

Comment:
This should probably be `dfmt: $O/dfmt.stamp`. But I suggest using `lint: $O/dfmt.stamp`, in case we can add more style checkers / linters in the future...